packages/genesys-spark-components/src/components/stable/gux-icon/gux-icon.scss

```diff
@@ -4,6 +4,21 @@
   height: 1em;
 }
 
+:host([size='small']) {
+  width: var(--gse-ui-icon-size-sm);
+  height: var(--gse-ui-icon-size-sm);
+}
+
+:host([size='medium']) {
+  width: var(--gse-ui-icon-size-md);
+  height: var(--gse-ui-icon-size-md);
+}
+
+:host([size='large']) {
+  width: var(--gse-ui-icon-size-lg);
+  height: var(--gse-ui-icon-size-lg);
+}
+
```

Comment:
Do we need to handle `inherit` here?

Reply:
no that is the default behaviour so no need for it here